generators/app/templates/config/protractor.conf.ts

```diff
@@ -10,11 +10,12 @@ let conf: Config = {
     '../specs/**/*.spec.js',
     '../specs/*.spec.js'
   ],
-  baseUrl: '<%=baseUrl%>',
+  baseUrl: 'http://build-a-list-test.com/',
```

Comment:
Please revert this. Yeoman Generator will put project url here, depending on what was passed as parameters